cmake/modules/FindOpenSP.cmake

```diff
@@ -1,4 +1,3 @@
-# SPDX-FileCopyrightText: 2009 Cristian Oneț <onet.cristian@gmail.com>
 # SPDX-FileCopyrightText: 2022 Dawid Wróbel <me@dawidwrobel.com>
```

Comment:
Still needs a license statement. If you're OK with GPL2-or-later just use 
```
/***************************************************************************
 *                                                                         *
 *   This program is free software; you can redistribute it and/or modify  *
 *   it under the terms of the GNU General Public License as published by  *
 *   the Free Software Foundation; either version 2 of the License, or     *
 *   (at your option) any later version.                                   *
 *                                                                         *
 ***************************************************************************/
```
(Lifted from context.cpp).

Reply:
BSD 3 for compatibility with CMake (where this module was filed as a PR).

Reply:
You can license it as GPL here and as BSD 3-clause in Cmake if you want to. It's your work. Regardless you need to say here what license applies and if it's not the GPL V2 you need to add a copy of the other license in the top source directory.